### PR TITLE
Show preview of the selected theme

### PR DESCRIPTION
**Pull request by NEBANU**
Solves issue 10 in the JabRef Themes Repo: https://github.com/JabRef/themes.jabref.org/issues/10
It added a preview of the light and dark theme when the user selects it.
The preview is removed if the user does not change their theme or chooses a custom them.
Dark Theme Preview:
image
Light Theme Preview:
image

### Mandatory checks

- [x] I own the copyright of the code submitted and I licence it under the [MIT license](https://github.com/JabRef/jabref/blob/main/LICENSE)
- [X] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if change is visible to the user)
- [X] Tests created for changes (if applicable)
- [X] Manually tested changed features in running JabRef (always required)
- [X] Screenshots added in PR description (for UI changes)
- [X] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [X] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Note that this project **does not** adhere to [Semantic Versioning](https://semv
 ## [Unreleased]
 
 ### Added
-
+- We added a preview for themes, when they are selected in preferences (to solve: https://github.com/JabRef/themes.jabref.org/issues/10)
```
Comment (koppor) [inline CHANGELOG.md:13]: Wrong format.

**Review by koppor — CHANGES_REQUESTED** [review]
Remove AI generated comments.

Remove image (not linked in the docs)

**Comment by Siedlerchr** [comment]
Does not solve the original issue and does not work for custom themes, although I like the idea

**Comment by koppor** [comment]
Closing this issue due to inactivity :zzz:

**Pull request closed without merging**